release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2376,6 +2376,8 @@ In the following tables, features are marked with the following statuses:
 
 * In {product-version} with IPv6 networking running on {rh-openstack-first} environments, `IngressController` objects configured with the `endpointPublishingStrategy.type=LoadBalancerService` YAML attribute will not function correctly. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2263550[*BZ#2263550*], link:https://bugzilla.redhat.com/show_bug.cgi?id=2263552[*BZ#2263552*])
 
+* In {product-version} with IPv6 networking running on {rh-openstack-first} environments, IPv6 load balancers are not allowed to be shared. (link:https://issues.redhat.com/browse/OCPBUGS-29605[*OCPBUGS-29605*])
```

Comment:
Could say a little bit more why this is a limitation or the consequence of it, please. For example, line2373 offers users why the limitation exists.

Reply:
done